numpy/polynomial/_polybase.py

```diff
@@ -67,6 +67,42 @@ class ABCPolyBase(abc.ABC):
     # Limit runaway size. T_n^m has degree n*m
     maxpower = 100
 
+    # Unicode character mappings for improved __str__
+    _superscript_mapping = {
+        0 : "\u2070",
+        1 : "\u00B9",
+        2 : "\u00B2",
+        3 : "\u00B3",
+        4 : "\u2074",
+        5 : "\u2075",
+        6 : "\u2076",
+        7 : "\u2077",
+        8 : "\u2078",
+        9 : "\u2079"
+    }
+    _subscript_mapping = {
+        0 : "\u2080",
+        1 : "\u2081",
+        2 : "\u2082",
+        3 : "\u2083",
+        4 : "\u2084",
+        5 : "\u2085",
+        6 : "\u2086",
+        7 : "\u2087",
+        8 : "\u2088",
+        9 : "\u2089"
```

Comment:
If you're not going to include the character directly, can you use the `\N` escapes which are more readable?

Reply:
Good point - including the characters directly is probably the most readable, so I've made that change in 380c9f0d3